manifests/state-spectrum-x-operator/0010-spectrumx-operator.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: spectrum-x-operator
+  namespace: {{ .RuntimeSpec.Namespace }}
+  labels:
+    control-plane: spectrum-x-operator
+    app.kubernetes.io/name: spectrum-x-operator
+spec:
+  selector:
+    matchLabels:
+      control-plane: spectrum-x-operator
+  replicas: 1
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: spectrum-x-operator
```

Comment:
This should match the name of the container

Reply:
done